src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -43,8 +54,26 @@ public class Function
         }
 
         /// <summary>
+        /// A flag used during serialization to indicate
+        /// that the node is a custom node.
+        /// </summary>
+        public bool IsCustomNode
```

Comment:
API Change : `IsCustomNode` and `FunctionUuid` properties are added to `Function`.